### ISSUE TITLE
Add early stopping

### ISSUE THREAD

**Issue by rcurrie** [rcurrie/tumornormal#3]
The current network runs through all 8 epochs with accuracy hovering in the 95% range. Should really use early stopping to speed up testing new models. Bonus to add some training visualization. 

For example:

http://parneetk.github.io/blog/neural-networks-in-keras/
